public/test/jest-setup.ts

```diff
@@ -75,7 +75,8 @@ jest.mock('../app/core/core', () => ({
   appEvents: testAppEvents,
 }));
 jest.mock('../app/angular/partials', () => ({}));
-jest.mock('../app/features/plugins/plugin_loader', () => ({}));
+// Mocking this means we can't test plugin loading/dynamic import of datasource components 🤷‍♂️
+// jest.mock('../app/features/plugins/plugin_loader', () => ({}));
```

Comment:
Mocking this meant that the tests wouldn't dynamically import the plugins/components required to render the Loki query editor - need to investigate removing it/see what the effect is/potentially just unmock it in the tests that need it